cinema/permissions.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import permissions
+
+
+class IsAdminOrIfAuthenticatedReadOnly(permissions.BasePermission):
+    def has_permission(self, request, view):
+        return request.user.is_authenticated \
+            if request.method in permissions.SAFE_METHODS\
+            else request.user.is_staff
```

Comment:
The line continuation character `\` is not needed here. Python automatically joins lines inside parentheses, brackets, and braces. You can remove the backslashes and enclose the entire expression in parentheses for better readability.